npm/package.json

```diff
@@ -7,6 +7,7 @@
     "copy-govuk-image-assets": "copyfiles -f node_modules/govuk-frontend/govuk/assets/images/* ../public/images",
     "copy-govuk-font-assets": "copyfiles -f node_modules/govuk-frontend/govuk/assets/fonts/* ../public/fonts",
     "copy-govuk-js-assets": "copyfiles -f node_modules/govuk-frontend/govuk/all.js ../public/javascripts",
+    "copy-tdr-image-assets": "copyfiles -f assets/images/*",
```

Comment:
For the header logo, Nigel's tried adding the image to assets/images directly: https://github.com/nationalarchives/tdr-transfer-frontend/pull/101/files

That might be easier in this case too.

Reply:
I tried this approach but for some reason the favicon wouldn't change, Tom also attempted it but didn't work. I had to change the file path within the `main.scala.html` file which worked!

Reply:
Sorry, I mean public/images. If you copy it to there, you shouldn't need the extra script in package.json.

Reply:
Yes that was the first thing I tried essentially to keep things simple but it didn’t work, it is the same thing Tom tried to do too but, I had to go through with it using the script approach eventually.

Reply:
@TomJKing Do you know why just putting the image in the folder doesn't work? It seems like it should be simpler than adding a script to the do the same thing.

Reply:
No idea. It seems that the favicon has some pretty hefty caching, which prevents easy updating